.github/dependabot.yml

```diff
@@ -23,6 +24,7 @@ updates:
     schedule:
       interval: "daily"
     labels:
+      - '6.16.z'
```

Comment:
Can we remove 6.12.z from here as well?

Reply:
@Gauravtalreja1 Lets not hold this PR for 6.12 changes, we can tackle that in a separate PR.

Reply:
@jyejare @Gauravtalreja1 updated

Reply:
@omkarkhatavkar Thank you!